canonical-web-and-design/charmhub.io
# Update details page

## Done

- Add JS for most tabs
- Add dummy data to channel map
- Added JS for channel map
- Updated some of the tabs with better fake data

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit http://0.0.0.0:8045
- Click around
- use the tabs
- change to different sections in the table of contents
- Refresh the page
- Use the browser forward and back buttons

## Issue / Card

Fixes #79
